test/helpers/organization_helpers.rb

```diff
@@ -0,0 +1,15 @@
+module OrganizationHelpers
```

Comment:
Why not `test/functional/helpers/organization_helpers.rb`? I think we're considering API expectations functional tests, and that file already exists. I don't see this helper being used in the unit tests.